### DISCUSSION TITLE
Manually Adding Pool

### DISCUSSION THREAD

**Discussion by ghost** [guydavis/machinaris#479] [Q&A]
Hi wallet sync is pretty damn slow. I saved my whole pool config from space pool. I there a way to plug in the codes without having to wait for nft?

this is what i have
Sync status: Synced
Wallet id 2:
Current state: FARMING_TO_POOL
Current state from block height: 537069
Launcher ID: EDITED FOR POSTING
Target address (not for plotting): EDITED FOR POSTING
Owner public key: EDITED FOR POSTING
P2 singleton address (pool contract address for plotting): EDITED FOR POSTING
Current pool URL: https://na1.pool.space
Current difficulty: None
Points balance: 0
Relative lock height: 64 blocks
Payout instructions (pool will pay to this address): EDITED FOR POSTING

**Comment by guydavis** [comment]
Sorry, not that I know of.  Machinaris WebUI is simply serving up the Chia CLI behaviors and this is how Chia official pooling works.  

References:
- https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference
- https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-FAQ
- https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide

Please mark this Discussion as Answered if it addressed your question.  Thanks!
